### ISSUE TITLE
End-to-end encryption still experimental?

### ISSUE THREAD

**Issue by eibex** [nextcloud/ios#1418]
In the settings of the app E2EE is still marked as experimental while the Nextcloud server app says it's production ready now. 

Is it still not stable on iOS or just a left over from the previous builds?

**Comment by gc0c5f** [comment]
Would be interested in this answer, too. Thank you @marinofaggiana! 😊

**Comment by marinofaggiana** [comment]
yes yes sorry !! too many task ... it's in experimental until I detect new issues, please your tests are highly appreciated for "remove" the experimental mode but the protocol and the encryption is solid and proven

**Comment by eibex** [comment]
First and foremost do not worry and thanks for your work, it is really appreciated.

I've been testing the app with encryption enable in a production environment without issues for almost a full month. 

I will open new issues if any bug arises.

**Comment by gc0c5f** [comment]
Thank you so much for taking the time and all of your hard work. 

If the encryption in the app and server connectivity are solid would it be safe to assume that there is no real chance of data corruption or data loss occurring? Is this the case? 

An "experimental" user experience like missing functionality or occasional crashes (without the chance for data loss!) might not be much a problem for interested users ^^

Thank you, again! ☺️
